lutris/installer/interpreter.py

```diff
@@ -532,6 +532,18 @@ def _map_command(self, command_data):
     # ----------------
 
     def _finish_install(self):
+        exe = self.script.get('game').get('exe')
```

Comment:
if the script doesn't have a 'game' section, this will crash.

Reply:
there are other ways to provide the 'exe', sometimes it's referred to as 'iso', 'main_file', 'rom'. There is code in the interpreter that handles that, maybe it should be refactored and used here as well.

Reply:
I tested it out with an installer without a game section, this didn't seem to cause a crash, however, I agree it could do with changing. As for the other ways to refer to the executable, I'll take another look and take this in to account